pandas/core/internals.py

```diff
@@ -2020,6 +2020,14 @@ def re_replacer(s):
 
         return block
 
+    def to_native_types(self, slicer=None, na_rep='nan', quoting=None,
+                        bytes_encoding=None, **kwargs):
```

Comment:
I put it in `ObjectBlock` because that's the only class that could contain objects--correct?  Given that a precondition of the code I'm adding is `dtype==object`, are you sure you'd want the logic to go in the base class for all types (even if it's overridden by some)?